adapters/iqzone/iqzone_test.go

```diff
@@ -0,0 +1,10 @@
+package iqzone
+
+import (
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"testing"
+)
+
+func TestJsonSamples(t *testing.T) {
+	adapterstest.RunJSONBidderTest(t, "iqzonetest", NewIQZoneBidder("http://nep.advangelists.com/xp/get?pubid={{.PublisherID}}"))
```

Comment:
Does IQZone use the Advangelists servers? .. or is this a copy/paste oversight?

Reply:
yes they do.

Reply:
Is there a reason to have a dedicated adapter then rather than simply declaring an alias of the Advangelists adapter? It would mean we would only have one adapter to maintain rather than two.

Reply:
Thanks @hhhjort. It's part of our client and tech need to get a new adapter for them.

Reply:
How is the processing different between this adapter and the Advangelists one? The alias system was developed just for this use case, to avoid duplicate work. So I would like to understand what is missing to make it work for you.

Reply:
Hi, Processing in both adapters is same at this moment. We wanted to have a different adapter name for our customer.  What changes are needed for implementing alias, do i have to update advangelists adapter.

Reply:
An alias is done on the adapter side, or potentially the PBS host side. See [Auction Docs](https://github.com/prebid/prebid-server/blob/master/docs/endpoints/openrtb2/auction.md) , the section on Aliases. A PBS host can define a default request that gets merged with all incoming requests. Any aliases defined there will be available for any publishers to use without them having to declare them in their request.

Reply:
@hhhjort 
PBS host - It seems all of the config is needed on server. I do not see any change in adapter. 
on Adapter - Is there any default request concept too, or a doc to look at what changes are needed.

Reply:
All the config is done on the PBS host, no code change is needed. This is the default request documentation: https://github.com/prebid/prebid-server/blob/master/docs/developers/default-request.md